internal/workload/podman/podman.go

```diff
@@ -30,13 +32,36 @@ const (
 	podmanStart  = string(podmanEvents.Start)
 	podmanRemove = string(podmanEvents.Remove)
 	podmanStop   = string(podmanEvents.Stop)
+
+	podmanBinary                  = "/usr/bin/podman"
+	autoUpdateServiceUnitTemplate = `[Unit]
+Description=Podman {{ .PodName }}.service
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+Restart=on-failure
+RestartSec=15
+ExecStart={{ .PodmanBinary }} play kube --replace {{ .ManifestPath }}
+ExecStop={{ .PodmanBinary }} play kube --down {{ .ManifestPath }}
+Type=forking
+
+[Install]
+WantedBy=default.target
+`
```

Comment:
I'm wondering if a template file can be easier to modify/maintain or if there is a reason to have this inline.

Reply:
Not sure, podman uses inline code as well: https://github.com/containers/podman/blob/main/pkg/systemd/generate/common.go#L26